README.md

```diff
@@ -172,6 +174,38 @@ The `pm-x` generator generates commands for running a package through JavaScript
 - `name`: The package name (by default tries to read from the `name` field in `package.json`).
 - `usage`: An additional string appended at the end of each command suggesting usage. (defaults to `""`).
 
+### `badges`
+
+The `badges` generator generates badges for npm version, npm downloads and some optional ones like codecov and bundle.
+
+#### Usage
+
+    <!-- automd:badges name=package-name -->
+    <!-- /automd -->
+
+**Updated Result:**
+
+    <!-- automd:badges name=package-name -->
+
+    [![npm version](https://img.shields.io/npm/v/package-name?style=flat&colorA=18181B&colorB=F0DB4F)](https://npmjs.com/package/package-name)
+    [![npm downloads](https://img.shields.io/npm/dm/package-name?style=flat&colorA=18181B&colorB=F0DB4F)](https://npmjs.com/package/package-name)
+
+    <!-- /automd -->
+
+#### Arguments
+
+- `name`: The package name (by default tries to read from the `name` field in `package.json`).
+- `style`: The badge style (defaults to `flat`).
+- `colorA`: The left color of the badge (defaults to `18181B`).
+- `colorB`: The right color of the badge (defaults to `F0DB4F`).
+- `downloads-enabled`: Show npm downloads badge (defaults to `true`).
+- `version-enabled`: Show npm version badge (defaults to `true`).
+- `codecov-enabled`: Show codecov badge (defaults to `false`).
+- `bundle-enabled`: Show bundle size badge (defaults to `false`).
+
+> [!NOTE]
+> Each badge `downloads|version|codecov|bundle` can be enabled or disabled using `*-enabled` arguments also can be overridden with custom URLs using `*-src`, `*-href` and badge name `*-name` arguments.
```

Comment:
Probably would be good to give an example of this here.

Reply:
Should be good to go 👍🏼